src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,4 +1,18 @@
-// Add the required types and props
-export const UserInfo = () => (
-  <>UserInfo markup</>
-);
+import React from 'react';
+import { User } from '../../types/User';
+
+type Props = {
+  user: User;
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => {
+  const { name, email } = user;
```

Comment:
If you need only a few properties you can destructurize them right there
```suggestion
export const UserInfo: React.FC<Props> = ({ user: { name, email } }) => {
```